fe/fe-core/src/main/java/org/apache/doris/nereids/memo/Group.java

```diff
@@ -184,7 +184,7 @@ public void setExplored(boolean explored) {
      * @return {@link Optional} of cost and {@link GroupExpression} of physical plan pair.
      */
     public Optional<Pair<Double, GroupExpression>> getLowestCostPlan(PhysicalProperties physicalProperties) {
-        if (physicalProperties == null || lowestCostPlans.isEmpty()) {
+        if (physicalProperties == null || lowestCostPlans == null || lowestCostPlans.isEmpty()) {
```

Comment:
The lowestCostPlans hash been initialed in Line.49. So the lowest cost plans could not be null.

Reply:
Well, the latest version has initialized it